README.md

```diff
@@ -180,6 +188,30 @@ To build the x64 target (recommended):
 3. `cmake --preset x64`
 4. `cmake --build --preset x64`
 
+To build the ARM target:
+
+1. Open an "x64 Native Tools Command Prompt for VS 2022 Preview".
+2. Change directories to the previously cloned `STL` directory.
+3. `"%VSINSTALLDIR%VC\Auxiliary\Build\vcvarsall.bat" amd64_arm`

```

Comment:
vcvarsall.bat understands `x64_MEOW` as a synonym, and we try to prefer the vendor-neutral and less-confusing x64 whenever possible.